node/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:latest 
+WORKDIR /app
+COPY . /app/
+
+# Copy package.json and package-lock.json (if available)
+COPY package*.json ./
```

Comment:
Instead of Copying the package.json files, do a clean installation using the same commands you run locally to run your application.